src/infra/criptography/jwt-adapter.ts

```diff
@@ -0,0 +1,14 @@
+import { type Encrypter } from '@/data/protocols'
+import jwt from 'jsonwebtoken'
+
+export class JwtAdapter implements Encrypter {
+  private readonly secret: string
+
+  constructor (secret: string) {
+    this.secret = secret
+  }
+
+  async encrypt (value: string): Promise<string> {
```

Comment:
We also implemented this logic in login task.

Reply:
Do you think I should copy your logic then?